objmodel/chapter.md

```diff
@@ -1,86 +1,90 @@
-
+<!-- English spelling: "behaviour" line 46 -->
 A Simple Object Model
 ======================
 
 
-Introduction
-----------------
+## Introduction
 
 Object-oriented programming is one of the major programming paradigms in use
-today with a lot of languages providing some form of object-orientation. While on the
+today, with a lot of languages providing some form of object-orientation. While on the
 surface the mechanisms that different object-oriented programming languages
 provide to the programmer are very similar, the details can vary a lot.
 Commonalities of most languages are the presence of objects and some kind of
 inheritance mechanism. Classes, however, are a feature that not every language
-supports directly. For example in prototype based languages like Self or
-JavaScript the concept of class does not exist and objects instead inherit
+supports directly. For example, in prototype-based languages like Self or
+JavaScript, the concept of class does not exist and objects instead inherit
 directly from each other.
 
 Understanding the differences between different object models can be
-interesting. It often reveals the family resemblance between different
-languages. It can be useful to quickly understand the model of a new language by
-putting it into the context of the models of other languages and it can give a
-better feeling of the programming language design space.
+interesting. They often reveal the family resemblance between different
+languages. 
+It can be useful to put the model of a new language into 
+the context of the models of other languages, 
+both to quickly understand the new model, and to get a
+better feeling for the programming language design space.
```

Comment:
Sometimes I rearranged your sentences for clarity and conciseness -- please read my changes carefully and let me know if I have changed your meaning by mistake!